indexing-solr/src/test/java/edu/unc/lib/boxc/indexing/solr/filter/SetDescriptiveMetadataFilterTest.java

```diff
@@ -218,6 +219,9 @@ public void testNamePartConcatenation() throws Exception {
         assertTrue(idb.getCreator().contains("Repo2, Boxy"));
         assertTrue(idb.getCreator().contains("Gilmer, Jeremy Francis, 1918-2020"));
         assertTrue(idb.getCreator().contains("Boxy, Berry Jean, Jr., 1991-"));
+        assertTrue(idb.getCreator().contains("Given"));
```

Comment:
Instead of checking for negative buggy cases, you could instead check the size of idb.getCreator() to make sure there are only exactly as many creator values as are expected